### ISSUE TITLE
[Nuxt-SSR] Query.with() using string[] throws `TypeError: name.split is not a function`

### ISSUE THREAD

**Issue by Eidher** [vuex-orm/vuex-orm#472]
When Nux (SSR) tries to execute this query on the server, it throws the following error ` [Vue warn]: Error in render: "TypeError: name.split is not a function" `

```javascript
    return this.$store.getters['entities/user/query']()
    .where(item => {
      return item.type === 'EMPLOYEE' || item.type === 'DIRECTOR'
    })
    .with(['memberships', 'phones'])
    .get();
```
The error occurs here:
- https://github.com/vuex-orm/vuex-orm/blob/5ee51b656d10d27bc02d1e335be17a6575d2ebe9/src/query/loaders/Loader.ts#L19

I believe since it's executed on the server they don't share the same Array prototype.

```javascript

// A solution could be replacing: 
if (name instanceof Array)

// with 
if (Array.isArray(name))

//  or any polyfill for not supported browsers like:
function isArray(o) {
  return Object.prototype.toString.call(o) === '[object Array]';
}
```

and Thanks for this amazing tool!

**Comment by kiaking** [comment]
@Eidher Ah... thank you so much for the report! Yeah I guess we should do that!

**Comment by Eidher** [comment]
Thank you

**Comment by kiaking** [comment]
Ah wait, I've created the PR but just not merged yet. I'll reopen this, and close when there is new version released!

**Comment by kiaking** [comment]
The issue has been solved at Vuex ORM 0.32.5! Thank you so much for the report!

**Comment by cuebit** [comment]
Resolved #473
